.github/workflows/ios-release-to-org-openfoodfacts-scanner.yml

```diff
@@ -74,7 +75,7 @@ jobs:
           VERSION_NAME: ${{ env.RELEASE_VERSION }}
 
       - name: Build app
-        run: cd ./packages/smooth_app && flutter build ios --release --no-codesign
+        run: cd ./packages/smooth_app && pod update Sentry && flutter build ios --release --no-codesign
```

Comment:
Probably `ios/pod` instead.

Reply:
unsure what you mean

Reply:
I mean `pod` must be run from the `ios` subfolder.